src/default-config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't really see the point of this file. Won't it be a huge risk of breaking changes if we want to upgrade the default configuration (or even add a new entry ?)

\+ the "command" cause a problem relative to my previous comment

Reply:
Good point, so, an empty file then? Or a file with an empty json object ?

Reply:
Ok, I'm still not used to github comments, I resolved a conversation although it was not finished. I agree that allowing any command would be best